.github/workflows/image.yml

```diff
@@ -0,0 +1,37 @@
+name: image
+
+on:
+  schedule:
+    - cron: '30 2 * * *'
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Looks like this workflow will also build pull requests - but does not push those builds to docker hub.

As it stands, it appears that:
 - Merge to master triggers the workflow which builds image, tags it, and pushes to docker hub
 - PR triggers the workflow which builds the image, tags it, but does not push it
 - Schedule just builds `master` @ 02:30 every day, acts same as push to master AFAIK

Reply:
Sending a PR before you know the tests actually pass IMO doesn't make sense. And in this particular case, it isn't like it will rack up a hefty bill or something.

Reply:
`pull_request`, by default (I think!), will also act on resyncs etc, so if the original branch is pushed to - it will run the tests again in the context of the PR

Reply:
I know for a fact that it does work like that, yet it doesn't really change anything. I think we've agreed with Adam over chat that this workflow will run on each push, so now we just have to wait for him to implement it.

Reply:
The CI workflow now runs on every push and PR does not have to be opened for it to run.